src/scene/Entity.cpp

```diff
@@ -27,8 +27,10 @@ Entity::Entity(std::shared_ptr<Mesh> mesh) : mesh(std::move(mesh)) {}
 
 void Entity::setTransform(Mat3x4f newTransform)
 {
-	transform = newTransform;
-	Scene::instance().requestASRebuild();
+	formerTransform = transform;
+	transform = {newTransform, Scene::instance().getTime()};
```

Comment:
1. What if the plugin updates transforms first, then sets time on the scene (or mixes up these operations depending on callbacks order)? I'd specify our requirements in the documentation (`core.h` file?)
2. I am considering renaming `transform` into `transformInfo` because it is no longer just transform.

Reply:
Done.